app/avatar/templates/clothing/spacecadet.txt

```diff
@@ -0,0 +1 @@
-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M612,1418l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M653,1376l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M695,1335l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M737,1293l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M778,1251l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M653,1460l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M695,1418l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M737,1376l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M778,1335l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M820,1293l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M737,1460l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M778,1418l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M820,1376l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M862,1335l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/></g><path d="M208,1254l-9,142m492-142,5,142" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="12.52"/><path d="M450,1079a469,469,0,0,1,184,38l117,50a156,156,0,0,1,95,143v79H54v-79a156,156,0,0,1,95-143l117-50a469,469,0,0,1,184-38Zm0-27a497,497,0,0,0-195,40l-117,50A183,183,0,0,0,27,1310v106H872V1310a183,183,0,0,0-111-168l-117-50a496,496,0,0,0-195-40Z" transform="translate(60 -829)" fill="#333"/><path d="M548,1084c0,1,0,20,0,21,0,54-45,97-102,97s-102-44-102-97v-21" transform="translate(60 -829)" fill="none" stroke="#333" stroke-width="11.05" opacity="0.15" style="isolation:isolate"/><polygon points="446 571 333 261 100 124 242 261 72 264 347 572 446 571" fill="#ffcc3b" stroke="#333" stroke-miterlimit="10" stroke-width="26.97"/><polygon points="574 570 687 261 920 124 778 261 948 264 673 572 574 570 574 570" fill="#ffcc3b" stroke="#333" stroke-miterlimit="10" stroke-width="26.97"/></g></svg>
```

Comment:
Can you modify the fill colors in this `.txt` to take primary_color similarly to the existing `.txt` templates?